File: test/fixtures/output/node/request/multipart-form-data.js
const request = require('request');

const options = {
  method: 'POST',
  url: 'https://httpbin.org/anything',
  headers: {'Content-Type': 'multipart/form-data; boundary=---011000010111000001101001'},
  formData: {foo: 'bar'}
};

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

File: __tests__/__fixtures__/output/csharp/httpclient/multipart-form-data.cs
var client = new HttpClient();
var request = new HttpRequestMessage
{
    Method = HttpMethod.Post,
    RequestUri = new Uri("https://httpbin.org/anything"),
    Content = new MultipartFormDataContent
    {
        new StringContent("bar")
        {
            Headers =
            {
                ContentDisposition = new ContentDispositionHeaderValue("form-data")
                {
                    Name = "foo",
                }
            }
        },
    },
};
using (var response = await client.SendAsync(request))
{
    response.EnsureSuccessStatusCode();
    var body = await response.Content.ReadAsStringAsync();
    Console.WriteLine(body);
}
